test/evergreen.yml

```diff
@@ -5535,6 +5538,70 @@ buildvariants:
   tasks:
     - name: examples-c-tsan
 
+# This variant runs all possible tasks under TSan but instructs TSan to not report any issues it
+# encounters. This allows us to check whether a TSan change may cause other unexpected failures
+# while also not having to fix every TSan issue discovered. At the conclusion of the TSan work this
+# variant will be deleted.
```

Comment:
It sounds like tests in this build variant would never fail. If that's the case, is it meaningful to schedule it as part of mainline builds? Will patch build enablement be good enough?  

> At the conclusion of the TSan work this variant will be deleted.

Can you share more about what "TSan work" refers to, is it actually the SPM-3221 project)?

Reply:
Yep patch builds will be sufficient for now given our updated Q3 priorities.

The motivation for this variant is [here](https://jira.mongodb.org/browse/WT-13166?focusedCommentId=6466728&focusedId=6466728&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6466728). The quick summary is we added some `#ifdef TSAN_BUILD` code that produced an error we would have caught if we ran the right tests under TSan. 
Luke and I have been discussing the right cadence for this and settled on once a day to try catch rare failures, but as mentioned above Q3 changes make `patch_only feasible`

 > Can you share more about what "TSan work" refers to, is it actually the SPM-3221 project)?

Yes this is the SPM-3221 work. When we add tests to the `ubuntu2004-tsan` variant we update WiredTiger to address the warnings, and then that test can be removed from `ubuntu2004-tsan-no-throw`.

Reply:
Thanks for the answers, @ajmorton. I agree using [patch_only](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#limiting-when-a-task-or-variant-will-run) can be the way to go.

Reply:
Updated in [931093e](https://github.com/wiredtiger/wiredtiger/pull/10812/commits/931093e4aa4b11dea7fe77cf03e0dcdc90b3ae87) 👍